diff --git a/node_modules/vite/dist/node/chunks/config.js b/node_modules/vite/dist/node/chunks/config.js
index 3b4037b..c3afcc3 100644
--- a/node_modules/vite/dist/node/chunks/config.js
+++ b/node_modules/vite/dist/node/chunks/config.js
@@ -2441,7 +2441,7 @@ const requestQuerySplitRE = /\?(?!.*[/|}])/;
 const requestQueryMaybeEscapedSplitRE = /\\?\?(?!.*[/|}])/;
 const blankReplacer = (match) => " ".repeat(match.length);
 function getHash(text, length = 8) {
-	const h = crypto.hash("sha256", text, "hex").substring(0, length);
+	const h = crypto.createHash("sha256").update(text).digest("hex").substring(0, length);
 	if (length <= 64) return h;
 	return h.padEnd(length, "_");
 }
